rpc/transaction.go

```diff
@@ -21,3 +22,27 @@ type Transaction struct {
 	EntryPointSelector  *felt.Felt    `json:"entry_point_selector,omitempty"`
 	CompiledClassHash   *felt.Felt    `json:"compiled_class_hash,omitempty"`
 }
+
+type MsgToL1 struct {
+	To      common.Address `json:"to_address"`
+	Payload []*felt.Felt   `json:"payload"`
+}
+
+type Event struct {
+	From *felt.Felt   `json:"from_address"`
+	Keys []*felt.Felt `json:"keys"`
+	Data []*felt.Felt `json:"data"`
+}
+
+// https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L1871
+type TransactionReceipt struct {
+	Type            string     `json:"type"`
```

Comment:
We should probably also make this an `enum` https://github.com/starkware-libs/starknet-specs/blob/master/api/starknet_api_openrpc.json#L1979

Reply:
Thanks, done!